portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -23,8 +25,22 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
[Optional] I'm curious now that you've gotten me thinking about page counts and edge cases, what happens if you're on page 3, it has only one comment, and you delete it? Do you have a means of reloading the comments and getting the user to page 2?

Reply:
Hmm just checked that right now. Looks like I forgot to account for that edge case. I can work on that right now